assets/js/components/UserMenu.js

```diff
@@ -185,6 +188,13 @@ export default function UserMenu() {
 					aria-controls="user-menu"
 					aria-label={ __( 'Account', 'google-site-kit' ) }
 					tooltip
+					customizedTooltip={
+						<Fragment>
+							<strong>Google Account</strong>
+							<p>{ userFullName }</p>
+							<p>{ userEmail }</p>
```

Comment:
These probably shouldn't be full paragraphs, and the text here should be translated:

```suggestion
							<strong>{ __( 'Google Account' ) }</strong>
							<br />
							{ userFullName }
							<br />
							{ userEmail }
```

seems more appropriate to me.

Reply:
Have added two lines after the Google Account "heading" to separate it from the "content". That is just based on my eye for design - but can always iterate with UX in the future.